CMake/cdat_modules_extra/setup_runtime.sh.in

```
@@ -10,20 +10,21 @@ function cleanup {
 # which can lead to errors.
 if [ -n "${UVCDAT_SETUP_PATH}" ] ; then
   if [ "${UVCDAT_SETUP_PATH}" = "${install_prefix}" ] ; then
+    echo "Nothing to do since UVCDAT is already setup at: ${UVCDAT_SETUP_PATH}" 1>&2
     cleanup
     return 0
   else
-    echo "ERROR: UVCDAT setup was previously sourced at \"${UVCDAT_SETUP_PATH}\"" 1>&2
-    echo "ERROR: There is no need to run setup_runtime manually anymore." 1>&2
-    echo "ERROR: Open a new shell in order to use a different install location." 1>&2
+    echo "INFO: UVCDAT setup was previously sourced at: ${UVCDAT_SETUP_PATH}" 1>&2
+    echo "INFO: There is no need to run setup_runtime manually anymore." 1>&2
+    echo "INFO: Open a new shell in order to use a different install location." 1>&2
```

I disagree: the script failed to load the requested environment. It should return an error code.